columnflow/plotting/plot_functions_1d.py

```diff
@@ -298,3 +300,142 @@ def plot_cutflow(
     ax.set_xticklabels(xticklabels, rotation=45, ha="right")
 
     return fig, (ax,)
+
+
+def plot_profile(
+    hists: OrderedDict[od.Process, hist.Hist],
+    config_inst: od.Config,
+    category_inst: od.Category,
+    variable_insts: list[od.Variable],
+    style_config: dict | None = None,
+    density: bool | None = False,
+    yscale: str | None = "",
+    hide_errors: bool | None = None,
+    process_settings: dict | None = None,
+    variable_settings: dict | None = None,
+    skip_base_distribution: bool = False,
+    base_distribution_yscale: str = "linear",
+    skip_variations: bool = False,
+    **kwargs,
+) -> plt.Figure:
+    """
+    Takes 2-dimensional histograms as an input and profiles over the second axis.
+
+    This task adds two custom parameters, *skip_base_distribution* and *base_distribution_yscale*,
+    that can be selected on command-line via the --general-settings parameter
+
+    Exemplary task call:
+
+    .. code-block:: bash
```

Comment:
`code-block` needs to be followed by an empty line for sphinx to render this properly.